### PR TITLE
Add time trackers (ActivityWatch, TimeTagger)

### PR DESCRIPTION
**Pull request by almarklein**
This PR:
* Adds a new category for time trackers.
* Moves Kimai there (unchanged).
* Adds ActivityWatch.
* Adds TimeTagger.

Disclaimer: I'm the author of TimeTagger.

----

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1813,6 +1812,15 @@ _Related: [Task management & To-do lists](#task-management--to-do-lists), [Softw
 - [Zammad](https://zammad.org/) - Easy to use but powerful open-source support and ticketing system. ([Source Code](https://github.com/zammad/zammad)) `AGPL-3.0` `Ruby`
 
 
+### Time Trackers
+
+**[`^        back to top        ^`](#)**
+
+- [ActivityWatch](https://activitywatch.net) - An app that automatically tracks how you spend time on your devices. ([Source Code](https://github.com/ActivityWatch/activitywatch)) `MPL-2.0` `Python`
+- [Kimai](https://www.kimai.org/) - Kimai is a free & open source timetracker. It tracks work time and prints out a summary of your activities on demand. ([Demo](https://www.kimai.org/demo/), [Source Code](https://github.com/kevinpapst/kimai2/)) `MIT` `PHP`
+- [TimeTagger](https://timetagger.app) - An open source time-tracker based on an interactive timeline and powerful reporting. ([Demo](https://timetagger.app/app/demo), [Source Code](https://github.com/almarklein/timetagger)) `GPL-3.0`, `Python`
```
Comment (nodiscc) [inline README.md:1821]: - please remove the comma between license/language tags

**Review by nodiscc — CHANGES_REQUESTED** [review]
- please add an entry to the table of contents since we're adding a new section

**Comment by almarklein** [comment]
@nodiscc 👍  I made the requested changes (added to TOC, and removed the comma).

**Review by nodiscc — APPROVED** [review]
thanks

**Pull request merged**